app/services/notices/setup/determine-return-forms.service.js

```diff
@@ -32,8 +32,12 @@ async function go(session, recipients, eventId) {
 
       notifications.push({
         content: returnForm,
+        eventId,
+        licences: JSON.stringify([licenceRef]),
```

Comment:
Have you tried setting `licences` as a JSON attribute on the model (app/models/notification.model.js)?

> We may as well add `metadata` whilst we're at it!

```javascript
  static get jsonAttributes() {
    return ['licences', 'metadata', 'personalisation']
  }
```

Once added, it should then be you wouldn't need to stringify objects before persisting.

```suggestion
        licences: [licenceRef],
```

But I could be wrong!